src/utils/validate.js

```diff
@@ -0,0 +1,37 @@
+import { RACING_ERROR } from '../constants/constants.js';
+
+/**
+ * 자동차 이름 유효성 검사 함수
+ * @param {string} carName
+ * @throws [ERROR] 메시지
+ */
+export function validateCarName(carName) {
+  if (carName.length < RACING_ERROR.MIN_CAR_NAME)
+    throw new Error(RACING_ERROR.NAME_EMPTY_ERROR);
+
+  if (carName.length > RACING_ERROR.MAX_CAR_NAME) {
+    throw new Error(RACING_ERROR.NAME_LENGTH_ERROR);
+  }
+}
```

Comment:
혹시 길이 관련 처리는 하나로 묶어 작성해보는 것은 어떨까요?

```suggestion
export function validateCarName(carName) {
  if (carName.length < RACING_ERROR.MIN_CAR_NAME || carName.length > RACING_ERROR.MAX_CAR_NAME)
    throw new Error(RACING_ERROR.NAME_EMPTY_ERROR);
}
```